roles/build_foreman_proxy_image/meta/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+dependencies:
+  - role: set_build_facts
+  - role: create_workspace
```

Comment:
Given the logic of `create_workspace` is required by `build_image` I would consider combining the two or setting this as a dependency of `build_image`.

Reply:
3 steps are needed: 

1) create workspace (`create_workspace` role)
2) add Dockerfile and assets for build into workspace (image specific role)
3) build image in workspace (`build_image` role)

I could set `create_workspace` as a dependency of `foreman{_proxy}_image` roles, but I could not tie `create_workspace` together with `build_image` as I need one step in between.